src/notion-provider.ts

```diff
@@ -9,9 +9,25 @@ type NotionProviderOptions = {}
 
 function NotionProvider(this: any, options: NotionProviderOptions) {
   const seneca: any = this
-  const entityBuilder = this.export('provider/entityBuilder')
+
+  const makeUtils = this.export("provider/makeUtils")
+
+  const { makeUrl, getJSON, postJSON, deleteJSON, entityBuilder } = makeUtils({
+    name: "notion",
+    // url: options.url,
+  })
    
   seneca.message('sys:provider,provider:notion,get:info', get_info)
+
+  const makeConfig = (config?: any) =>
+    seneca.util.deep(
+      {
+        headers: {
```

Comment:
no longer needed?

Reply:
it is actually needed. We must use `fetch` as the sdk is incomplete